editor/inspector/components/label.js

```diff
@@ -161,8 +164,8 @@ exports.ready = function() {
                     const style = fontStyles[index];
                     const label = document.createElement('ui-label');
                     label.innerHTML = styleDisplayNames[index];
-                    setTooltip(label, this.dump.value[style]);
                     label.setAttribute('key', style);
+                    label.setAttribute('tooltip', this.dump.value[style].tooltip);
```

Comment:
Is this.dump.value[style].tooltip start with 'ENGINE. '

Reply:
the prefix 'ENGINE.' is recently added in the dump data